types/orchestrationTypes.d.ts

```diff
@@ -1,6 +1,6 @@
 import { FunctionOptions, FunctionTrigger, InvocationContext, LogHandler } from "@azure/functions";
-import { Task, TimerTask, TokenSource } from ".";
-import { EntityId, RetryOptions } from "../classes";
+import { RetryOptions, Task, TimerTask, TokenSource } from ".";
+import { EntityId } from "./entityTypes";
 
```

Comment:
While we're renaming, I'd prefer if we get rid of `Types` from these file names. Now that its in the `types/` folder and has the `.d.ts` extension, I think `Types` is implied

Reply:
Makes sense, done.